### PR TITLE
Post: Read the Docs loves Ruby

### LINKED ISSUE #255: Blog: Read the Docs :heart: Ruby
We should write a small blog post "Read the Docs :heart: Ruby" now that we added support for it. At least to communicate this is now supported and how to define it in the YAML file --also post from social. 

https://github.com/readthedocs/readthedocs-docker-images/pull/200/

### PR DESCRIPTION
**Pull request by humitos**
Initial content for the blog post.

✍🏼 Rendered version: https://read-the-docs-website--268.com.readthedocs.build/blog/2024/03/read-the-docs-loves-ruby/

Closes #255 

----
📚 Documentation preview 📚: https://read-the-docs-website--268.com.readthedocs.build/

### CODE REVIEW

File: content/posts/read-the-docs-loves-ruby.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+category: Feature announcement
+tags: builders, tools
+authors: Manuel Kaufmann
+status: published
+image: /images/read-the-docs-loves-ruby.jpg
+image_credit: Photo by <a href="https://unsplash.com/@jasondeblooisphotography?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Jason D</a> on <a href="https://unsplash.com/photos/selective-focus-photo-of-red-gemstone-VKLJ-BJlszE?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+We are happy to announce that **support for Ruby is now available** as an option for our pre-defined building tools ([`build.tools`](https://docs.readthedocs.io/en/stable/config-file/v2.html#build-tools)).
+Now, projects requiring Ruby to build their documentation are officially supported on Read the Docs 🎉
```
Comment (ericholscher) [inline content/posts/read-the-docs-loves-ruby.md:12]:
```suggestion
Projects requiring Ruby to build their documentation are officially supported on Read the Docs 🎉
```

File: content/posts/read-the-docs-loves-ruby.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Demo project using Jekyll
+
+As an example of a project using Ruby, [here you have a demo of the default Jekyll blog](https://test-builds.readthedocs.io/en/jekyll/) built and hosted on Read the Docs.
```
Comment (ericholscher) [inline content/posts/read-the-docs-loves-ruby.md:16]:
```suggestion
As an example of a project using Ruby, [see a demo of the default Jekyll blog](https://test-builds.readthedocs.io/en/jekyll/) built and hosted on Read the Docs.
```

File: content/posts/read-the-docs-loves-ruby.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+The following code is the `readthedocs.yaml` configuration file used to build it:
```
Comment (ericholscher) [inline content/posts/read-the-docs-loves-ruby.md:17]:
```suggestion
The following code is the `.readthedocs.yaml` configuration file used to build it:
```

File: content/posts/read-the-docs-loves-ruby.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+like [Jekyll](https://jekyllrb.com/), [Asciidoctor](https://asciidoctor.org/), [yard](https://yardoc.org/), [rdoc](https://ruby.github.io/rdoc/), and many others.
+
+
+## Try it out and give us feedback
+
+We encourage you to give Read the Docs a try if you are using any of these documentation tools,
+and take advantage of lot of extra features we provide that will empower your documentation workflow,
+like [pull request previews](https://docs.readthedocs.io/en/stable/pull-requests.html), among others.
+
+Please, [let us know](mailto:support@readthedocs.com) how was your experience with any of these Ruby documentation tools and if you have any issue in the process.
```
Comment (ericholscher) [inline content/posts/read-the-docs-loves-ruby.md:49]:
```suggestion
Please [let us know](mailto:support@readthedocs.com) how your experience is with any of these Ruby documentation tools, and how we might improve our support.
```

**Review by ericholscher — APPROVED** [review]
Short and sweet. This is great SEO/marketing content. We should probably also copy the demo & config over to a documentation page highlighting support for Jekyll or similar, but this is a great start. :100:

**Comment by humitos** [comment]
> We should probably also copy the demo & config over to a documentation page highlighting support for Jekyll or similar

Yeah, I thought about this and I immediately thought about the "example projects". However, I always felt the "example projects" are _too much_ for this use case. I would like to have something in between the `test-builds` branches (which are _too dev-y_) and the "example projects".

Maybe it's just a small documentation page per doctool where we define the `.readthedocs.yaml` file and explain _why_ we use some of those options. For example `--destination` and other parameters. @agjohnson has a similar question in https://github.com/readthedocs/website/pull/263#issuecomment-1949173267

**Comment by ericholscher** [comment]
Yeah, I'm thinking something very simple, like a tools index with pages per tool.

**Pull request merged**